assets/js/googlesitekit/modules/datastore/modules.js

```diff
@@ -206,11 +297,31 @@ const baseSelectors = {
 	 * @param {Object} state Data store's state.
 	 * @return {(Object|undefined)} Modules available on the site.
 	 */
-	getModules( state ) {
+	getModules: createRegistrySelector( ( select ) => ( state ) => {
 		const { modules } = state;
 
-		return modules;
-	},
+		// Return `undefined` if modules haven't been loaded yet.
+		if ( modules === undefined ) {
+			return undefined;
+		}
+
+		const registryKey = select( CORE_SITE_STORE_NAME ).getRegistryKey();
+		const sortedModules = sortBy( modules, [ ( { order } ) => order ] );
```

Comment:
Why use `lodash` over JS built-in `.sort()` methods? I find these methods offer little extra convenience but add to the cognitive load of reading the code. Lodash's API isn't universally known.

If there's a benefit to this then fine to keep, but if this could be accomplished with:

```suggestion
		const sortedModules = modules.sort((a, b) => {
			return a.object > b.object ? 1 : -1;
		});
```

Reply:
modules is an object so using sortBy makes the code a little more concise. I'll add a comment to the code to be more clear.